db/migrate/20140627193814_add_images_to_categories.rb

```diff
@@ -3,6 +3,7 @@
 class AddImagesToCategories < ActiveRecord::Migration[4.2]
   def change
     add_column :categories, :logo_url, :string
+    add_column :categories, :logo_dark_url, :string
```

Comment:
This won't work. In Rails, once a migration has been run on a database, it won't be run again. Thus, we can't add actions to existing migrations. All the changes that you need to make to the database structure need to go in a new migration file. 

This applies to all the changes under `db/migrations`. I think that for this PR, we just need to add a `uploaded_logo_dark` column in a new migration. 

See https://edgeguides.rubyonrails.org/active_record_migrations.html for a more in-depth look at Rails migrations.

Reply:
Well, seeing it now, what I did there was atrocious!
I reverted the changes and created a new migration file now.
I also had to add `uploaded_logo_dark_id` in it.